Makefile

```diff
@@ -67,7 +67,7 @@ endif
 build: build-ubuntu
 
 .PHONY: test
-test: golangci
+#test: golangci
```

Comment:
remember to uncomment this before opening the upstream PR

Reply:
btw, it is ok if you have linter errors when you submit the PR. The CI system will report the issues to you and you can address them. When my patch is merged (https://github.com/vmware-tanzu/antrea/pull/1001), you will also be able to run golangci-lint locally.

Reply:
remember to take care of this. you should rebase `upstream/master` now. I fixed the `golanci` target on macOS.